src/mz/src/ui.rs

```diff
@@ -65,6 +65,17 @@ impl OutputFormatter {
         }
     }
 
+    /// Outputs a prefix warning.
+    pub fn output_warning(&self, msg: &str) -> Result<(), Error> {
+        if self.no_color {
+            println!("\n* Warning * {}", msg);
```

Comment:
Nit: could we output these to stderr rather than stdout?

Reply:
Yes. A 100%. From now on, any warn goes to stdout.